ocs_ci/ocs/platform_nodes.py

```diff
@@ -2307,3 +2309,58 @@ def restart_nodes_by_stop_and_start_teardown(self):
         if stopped_vms:
             logger.info(f"The following VMs are powered off: {stopped_vms}")
             self.rhv.start_rhv_vms(stopped_vms)
+
+
+class IBMCloud(NodesBase):
+    """
+    IBM Cloud class
+    """
+
+    def __init__(self):
+        super(IBMCloud, self).__init__()
+        self.ibmcloud = ibmcloud.IBMCloud()
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
+        """
+        Restart all the ibmcloud vm instances
+
+        Args:
+            nodes (list): The OCS objects of the nodes instance
+            timeout (int): time in seconds to wait for node to reach 'not ready' state,
+                and 'ready' state.
+            wait (bool): True if need to wait till the restarted node reaches
+                READY state. False otherwise
+
+        """
+        self.ibmcloud.restart_nodes(nodes, timeout=900, wait=True)
+
+    def restart_nodes_by_stop_and_start(self, nodes, force=True):
+        """
+        Make sure all the nodes which are not ready on IBM Cloud
```

Comment:
This docstring seems to be not complete.

Reply:
Done

Reply:
I meant, that the sentence `Make sure all the nodes which are not ready on IBM Cloud` doesn't explain what to _make sure_.